### ISSUE TITLE
Red outline for selected line?

### ISSUE THREAD

**Issue by Tyriar** [Monokai/monokai-pro-vscode#1]
I'm seeing this red outline for all monokai pro themes on VS Code Insiders:

<img width="831" alt="screen shot 2017-08-21 at 9 48 14 am" src="https://user-images.githubusercontent.com/2193314/29529802-f2d9931c-8655-11e7-9da2-93069f7e645c.png">

**Comment by Tyriar** [comment]
This setting works around the issue:

```js
  "workbench.colorCustomizations": {
    "editor.lineHighlightBorder": "#00000000"
  }
```

**Comment by rebornix** [comment]
The only inconsistency I see is the borders don't work with editor setting `editor.renderLineHighlight`, which only takes background color setting from themes.

**Comment by bpasero** [comment]
I am not seeing this in stable, so it looks like an insiders only issue (which will then hit stable once we release):

![image](https://user-images.githubusercontent.com/900690/29557518-a515b4d8-8729-11e7-97c8-2d1deea1631f.png)

**Comment by Tyriar** [comment]
I also see #F00 on the message box background on Insiders:

![image](https://user-images.githubusercontent.com/2193314/29580134-d3c2f642-8729-11e7-80c5-a320de5a7214.png)

This repro's for me with the following settings.json:

```js
{
  "workbench.colorTheme": "Monokai Pro (Filter Spectrum)"
}
```

**Comment by roblourens** [comment]
I thought maybe this is the "free evaluation" part :)

**Comment by Monokai** [comment]
fixed in 1.1.0
